openadapt/app/tray.py

```diff
@@ -155,6 +169,25 @@ def __init__(self) -> None:
         # self.app_action.triggered.connect(self.show_app)
         # self.menu.addAction(self.app_action)
 
+        self.dashboard_action = TrackedQAction("Launch Dashboard")
+        self.dashboard_action.triggered.connect(self.launch_dashboard)
+        self.menu.addAction(self.dashboard_action)
+
+        self.download_button_text = (
+            f"Download Latest Version v{LATEST_VERSION}"
+            if Version(CURRENT_VERSION) < Version(LATEST_VERSION)
+            else "No updates available"
+        )
+        self.download_update_action = TrackedQAction(self.download_button_text)
```

Comment:
What do you think about disabling the menu item if no updates are available?

Reply:
Done